keyvi/src/cpp/dictionary/fsa/automata.h

```diff
@@ -40,6 +40,7 @@
 #include "dictionary/util/endian.h"
 #include "dictionary/fsa/internal/intrinsics.h"
 #include "dictionary/dictionary_merger_fwd.h"
+#include "dictionary/util/keyvi_file.h"
```

Comment:
so far we do not have the concept of a keyvi file yet. Anyway, this change basically makes keyvi_file a utility class for automata.